tests/ImplementationLoader.php

```diff
@@ -192,15 +195,21 @@ public function getRepository()
 
     public function getFixtureLoader()
     {
-        $testerClass = '\\Jackalope\\Test\\Tester\\' . ucfirst(strtolower($this->connection->getWrappedConnection()->getAttribute(PDO::ATTR_DRIVER_NAME)));
-        if (!class_exists($testerClass)) {
-            // load Generic Tester if no database specific Tester class found
-            $testerClass = Generic::class;
+        $platform = $this->connection->getDatabasePlatform();
+        switch (true) {
```

Comment:
```suggestion
        switch ($platform) {
```

semantically the same, but i find it nice to specify the thing that we switch on here